src/cladetime/cladetime.py

```diff
@@ -150,7 +149,7 @@ def sequence_metadata(self):
     @sequence_metadata.getter
     def sequence_metadata(self) -> pl.LazyFrame:
         """
-        :class:`polars.LazyFrame` : A Polars LazyFrame that references
+        :external+polars:std:doc:`polars.LazyFrame<reference/lazyframe/index>` : A Polars LazyFrame that references
```

Comment:
_Finally_ figured out how to link to documentation for other projects that use sphinx